File: backend/zego_server_assistant/token/go/sample/sample-base.go
package main

import (
    "fmt"
    "github.com/ZEGOCLOUD/zego_server_assistant/token/go/src/token04"
)

/*
Example code for generating basic authentication tokens
*/

func main() {
    // Please modify the appId to your own appId, which is a number obtained from the ZEGO console.
    // For example: 1234567890
    var appId uint32 = 1234567890

    // Please modify serverSecret to your own serverSecret, which is a string obtained from the ZEGO console.
    // For example: "fa94dd0f974cf2e293728a526b028271"
    serverSecret := ""

    // Please modify userId to the user's user_id.
    userId := "user1"

    // The validity period of the token, in seconds.
    var effectiveTimeInSeconds int64 = 3600
    // Token business authentication extension, leave it blank for basic authentication token.
    var payload string = ""

    // Generate the token.
    token, err := token04.GenerateToken04(appId, userId, serverSecret, effectiveTimeInSeconds, payload)
    if err != nil {
        fmt.Println(err)
        return
    }
    fmt.Println(token)
}

File: backend/zego_server_assistant/token/nodejs/sample/sample-base.js
/**
 * Example of generating basic authentication token.
 */

const { generateToken04 } = require('../server/zegoServerAssistant');

// Please modify appID to your own appId. appid is a number.
// Example: 1234567890
const appID = ; // type: number

// Please modify serverSecret to your own serverSecret. serverSecret is a string.
// Example: 'sdfsdfsd323sdfsdf'
const serverSecret = ;// type: 32 byte length string

// Please modify userId to the user's userId.
const userId = 'user1';// type: string

const effectiveTimeInSeconds = 3600; //type: number; unit: s; expiration time of token, in seconds.

// When generating a basic authentication token, the payload should be set to an empty string.
const payload = ''; 
// Build token 
const token =  generateToken04(appID, userId, serverSecret, effectiveTimeInSeconds, payload);
console.log('token:',token);
